.kokoro/presubmit/node12/common.cfg

```diff
@@ -22,3 +22,7 @@ env_vars: {
     key: "TRAMPOLINE_BUILD_FILE"
     value: "github/nodejs-retail/.kokoro/test.sh"
 }
+env_vars: {
+  key: "SECRET_MANAGER_KEYS"
```

Comment:
this file is generated from templates, so will be overwritten. To prevent this create an `owlbot.py` in the root directory with `.kokoro/presubmit/node12/common.cfg` in the `excludes` list.

Here's an example of a config file that ignores README.md (which is also generated).

https://github.com/googleapis/github-repo-automation/blob/main/owlbot.py#L17

Reply:
owlbot is added